messages/observations/heading.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Heading",
+    "description": "Reading from a heading sensor",
+    "required": [
+        "sensor_id",
+        "heading",
+        "type"
+    ],
+    "properties": {
+        "sensor_id": {
+            "type": "string",
+            "title": "Physical sensor id",
+            "desciption": "Id of the reporting physical sensor equipment"
+        },
+        "heading": {
+            "type": "number",
+            "title": "Heading",
+            "description": "Heading, relative to North [degrees]"
+        },
+        "type": {
+            "type": "string",
+            "title": "Type of sensor",
+            "enum": [
+                "Magnetic",
+                "True"
+            ]
+        }
+    }
+}
```

Comment:
Should we include for all the sensor som state message or should that be its own category

Reply:
Not sure what you mean? Like a `Status: Failed` ? Or do you have something else in mind?

Reply:
Example, the GPS gives a failure message or engines comparing over some abnormal sensor reading

Reply:
I will keep this out of the initial draft since we have too little knowledge about what and how we want to do. All observation message type are extensible, so in the meantime its perfectly fine for us to try any of the above suggestion out without adding it to the schema.